kvbc/CMakeLists.txt

```diff
@@ -1,7 +1,7 @@
 cmake_minimum_required (VERSION 3.2)
 project(libkvbc VERSION 0.1.0.0 LANGUAGES CXX)
 
-find_package(Boost ${MIN_BOOST_VERSION})
+find_package(Boost ${MIN_BOOST_VERSION} REQUIRED)
```

Comment:
I don't think boost actually is required for KVBC. We might be able to just remove this line altogether.

Reply:
Done